dll/win32/shell32/brfolder.cpp

```diff
@@ -343,6 +343,13 @@ BrFolder_InsertItem(
     _In_ PCIDLIST_ABSOLUTE pidlParent,
     _In_ HTREEITEM hParent)
 {
+    if (!(BrowseFlagsToSHCONTF(info->lpBrowseInfo->ulFlags) & SHCONTF_NONFOLDERS))
```

Comment:
```suggestion
    if (!(BrowseFlagsToSHCONTF(info->lpBrowseInfo->ulFlags) & SHCONTF_NONFOLDERS))
    {
```

Reply:
The next if block technically also has more than one line so according to your argument that should have braces as well?

Reply:
https://reactos.org/wiki/Coding_Style#Braces
Yes indeed, but I was hesitant to propose it because that if was inside a ifdef block